CHANGELOG.md

```diff
@@ -41,6 +41,9 @@
     - `axpby` (alias of `sapyb`)
     - `shape` setter in `DataContainer` and `ImageGeometry`
     - use of integer compression in `NEXUSDataWriter`
+  - Changes that break backwards compatibility:
+      - Moved fill methods in `geometry.allocate()` to `DataContainer.fill()`, now the `RANDOM` and `RANDOM_INT` methods can be accessed from `allocate` or `fill` (#2037)
```

Comment:
Does this break backward compatibility?
I guess the breaking change is the below one?